src/js/site/main.js

```diff
@@ -421,3 +421,21 @@ $(function () {
     });
   });
 });
+
+/*
+  Check if users have dark mode enabled
+  Set key if user response has not already been tracked to ensure we dont get multiple events from the same user
+ */
+$(document).ready(() => {
```

Comment:
dumb q but have you tried this without the document.ready? I'm unsure if we need it.

also to echo @teesloane, I think it would be preferrable to extract this into a unit testable function. and then call said function in app.js.

Reply:
You are correct sir! no `document.ready` needed. Ive imported the function to `app.js` where its being invoked